pkg/v1/tkg/tkgconfigupdater/helper.go

```diff
@@ -49,8 +49,15 @@ func unzip(srcfilepath, destdir string) error {
 	}
 	defer r.Close()
 
+	if strings.Contains(destdir, "..") {
+		return errors.Wrapf(err, "destination contains .. which could lead to zip file being extracted outside the current directory")
```

Comment:
nit : `destination contains ..` may be confusing , may be `destiation contains directory(parent) ".."`?